src/Symfony/Component/Security/Core/Authorization/Voter/VoterInterface.php

```diff
@@ -33,7 +33,7 @@ interface VoterInterface
      * @param mixed $subject    The subject to secure
      * @param array $attributes An array of attributes associated with the method being invoked
      *
-     * @return int either ACCESS_GRANTED, ACCESS_ABSTAIN, or ACCESS_DENIED
+     * @return int-mask-of<self::ACCESS_*> either ACCESS_GRANTED, ACCESS_ABSTAIN, or ACCESS_DENIED
```

Comment:
this is not true. It does not return an bitmask.
This can be document as `@return self::ACCESS_*` though